ownCloud/Client/SortBar.swift

```diff
@@ -73,6 +74,7 @@ class SortBar: UIView, Themeable {
 		sortButton.accessibilityIdentifier = "sort-bar.sortButton"
 
 		leftButtonImage = Theme.shared.image(for: "folder-create", size: CGSize(width: 30.0, height: 30.0))!.withRenderingMode(.alwaysTemplate)
```

Comment:
@pablocarmu Constant instead of plain value 30?

Reply:
Sure, I'll do it!